packages/phasor/src/elements/brush/brush-element.ts

```diff
@@ -0,0 +1,76 @@
+import { Utils } from '../../utils/tl-utils.js';
+import { Vec } from '../../utils/vec.js';
+import { deserializeXYWH, serializeXYWH } from '../../utils/xywh.js';
+import { BaseElement, HitTestOptions } from '../base-element.js';
+
+export class BrushElement extends BaseElement {
+  type = 'brush' as const;
+  color = '#000000';
+  points: number[][] = [];
+  lineWidth = 4;
+
+  get tiny() {
```

Comment:
It doesn't feel right to have an ad-hoc processing for `tiny`. We should expand the hit test area for picking element. For example, there is an `expandStroke` option in `HitTestOptions` that is prepared for this case, not used ATM though.

Reply:
`tiny` is not only for hit testing but also for rendering. When the pattern is too small, only draw a circle instead of whole paths.